.github/CODEOWNERS

```diff
@@ -30,7 +30,7 @@
 /integration-tests/tests/run-analysis.js @DataBiosphere/broad-interactive-analysis
 /integration-tests/tests/run-rstudio.js @DataBiosphere/broad-interactive-analysis
 /integration-tests/tests/analysis-context-bar.js @DataBiosphere/broad-interactive-analysis
-/integration-tests/tests/export-*-dataset-to-new-workspace.js @DataBiosphere/data-explorer-eng
+/integration-tests/tests/export-*-dataset-to-workspace.js @DataBiosphere/data-explorer-eng
```

Comment:
I noticed data-explorer-eng wasn't automatically requested for review. It looks like the codeowners configuration was missed when these tests were renamed in #4188.